packages/orchestration/src/proposals/start-stakeAtom.js

```diff
@@ -22,23 +24,32 @@ const trace = makeTracer('StartStakeAtom', true);
  *     };
  *   };
  * }} powers
+ * @param {{
+ *   options: {
+ *     chainInfo: Record<string, CosmosChainInfo>;
+ *     assetInfo: [Denom, DenomDetail & { brandKey?: string }][];
+ *   };
+ * }} config
  */
-export const startStakeAtom = async ({
-  consume: {
-    agoricNames,
-    board,
-    chainStorage,
-    chainTimerService: timer,
-    cosmosInterchainService,
-    startUpgradable,
-  },
-  installation: {
-    consume: { stakeIca },
-  },
-  instance: {
-    produce: { stakeAtom: produceInstance },
+export const startStakeAtom = async (
+  {
```

Comment:
odd whitespace change. makes this part of the diff a little harder to read

Reply:
The change here and in `startStakeOsmo`, `startStakeBld` is incorporating the second argument, `options`